app/views/drafts/show.html.erb

```diff
@@ -1,3 +1,8 @@
+<% content_for :header_title do %>
+  <h2><%= display_entry_id(get_resource.draft, resource_name) %></h2>
+  <p class="subtitle"><%= display_variable_long_name(get_resource.draft) %></p>
```

Comment:
This feels a little odd because it this is the generic draft show view, but you're using a method specific to variables. I'm not sure if that means we need to change it since collections has its own show view, but just wanted to note that it seems odd.